assets/js/stripe.js

```diff
@@ -528,7 +528,7 @@ jQuery( function( $ ) {
 				$( wc_stripe_form.form ).off( 'submit', wc_stripe_form.form.onSubmit );
 			}
 
-			wc_stripe_form.form.submit();
+			wc_stripe_form.form.trigger( 'submit' );
```

Comment:
Are you sure submit is being deprecated? I couldn't find that in the linked docs, nor on https://api.jquery.com/submit/

Reply:
None of these deprecations are documented. I used the list [here](https://github.com/jquery/jquery/blob/305f193aa57014dc7d8fa0739a3fefd47166cd44/src/event/alias.js#L10-L21) to figure out which shorthands are being deprecated.

There are more details on how I found the shorthands we use in [this comment](https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1390#issuecomment-738917808) 🙂 

For information about the deprecations, see [this jQuery issue](https://github.com/jquery/api.jquery.com/issues/972).